src/main/java/com/minecolonies/coremod/entity/ai/mobs/EntityAIBarbarianAttackMelee.java

```diff
@@ -0,0 +1,98 @@
+package com.minecolonies.coremod.entity.ai.mobs;
+
+import com.minecolonies.api.util.ItemStackUtils;
+import net.minecraft.entity.EntityCreature;
+import net.minecraft.entity.EntityLivingBase;
+import net.minecraft.entity.ai.EntityAIBase;
+import net.minecraft.init.SoundEvents;
+import net.minecraft.item.ItemStack;
+import net.minecraft.item.ItemSword;
+import net.minecraft.util.DamageSource;
+import net.minecraft.util.EnumHand;
+
+/**
+ * Created by Asher on 13/6/17.
+ */
+public class EntityAIBarbarianAttackMelee extends EntityAIBase
+{
+
+    final private EntityCreature   entity;
+    private       EntityLivingBase target;
+    private int LastAttack = 0;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this field "LastAttack" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS00116)